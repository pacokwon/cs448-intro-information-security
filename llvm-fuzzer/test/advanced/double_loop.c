<<<<<<< HEAD
int main()
{
    int ave = 0;
=======
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int ave = 0;
    char input[65536];

    fgets(input, sizeof(input), stdin);
>>>>>>> fuzzer/master

    for(int i = 1; i < 100; i++)
    {
        for(int j = 0; j < 100; j++)
        {
            int k = i + j;
            int w = i - j;

<<<<<<< HEAD
=======
            w = atoi(input) + 10;
>>>>>>> fuzzer/master
            int z = k / w;

            if(i >= 10)
            {
                int m = k / i;
            }
        }
    }
}
