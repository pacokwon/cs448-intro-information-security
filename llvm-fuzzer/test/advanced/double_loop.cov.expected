5,9
5,22
5,5
7,13
7,26
7,9
9,23
10,23
12,23
14,18
14,16
18,9
7,34
7,9
7,26
7,9
9,23
10,23
